code/modules/mob/living/simple_animal/abnormality/zayin/wellcheers.dm

```diff
@@ -59,9 +59,9 @@
 	return
 
 /mob/living/simple_animal/hostile/abnormality/wellcheers/work_chance(mob/living/carbon/human/user, chance)
-	return chance - success_counter
+	return chance - (success_counter * 2)
```

Comment:
Holy shit the real change. Shadow Nerfing Wellcheers

Reply:
qliphoth overload on a zayin

Reply:
Why are we nerfing wellcheers _again_